include/oboe/AudioStream.h

```diff
@@ -192,6 +266,15 @@ class AudioStream : public AudioStreamBase {
         return ResultWithValue<double>(Result::ErrorUnimplemented);
     }
 
+    /**
+     * Get the timestamp that the frame at `framePosition` was presented to the
+     * audio hardware.
```

Comment:
Should this be the time when the sound in air is produced?
Here is a definition:
https://developer.android.com/reference/android/media/AudioTimestamp

Reply:
I based the description on: https://developer.android.com/ndk/reference/group/audio#aaudiostream_gettimestamp

so we probably need to update those docs since "presented" isn't clearly defined (I always thought it was the time an audio frame was presented to, or received from, the codec) 

Description updated in accordance with `AudioTimestamp` docs